docker/NotebookImage.Dockerfile

```diff
@@ -1,5 +1,5 @@
 ARG python_version="3.9"
-FROM jupyter/base-notebook:python-${python_version}
+FROM quay.io/jupyter/base-notebook:python-${python_version}
 
 ARG python_version
 ARG arcgis_version="2.2.0"
```

Comment:
please update to 2.3.1

Reply:
beat you to it. these are defaults, and selectable in the GH actions UI.